.github/workflows/ci.yml

```
@@ -184,24 +184,24 @@ jobs:
             end
 
             File.open(ENV.fetch("GITHUB_ENV"), "a") do |f|
-              f.puts "CASK_CONFLICTS=\"#{cask_conflicts&.join(" ")}\"" if cask_conflicts.present?
-              f.puts "CASK_DEPENDENCIES=\"#{cask_dependencies&.join(" ")}\"" if cask_dependencies.present?
+              f.puts "CASK_CONFLICTS=#{cask_conflicts&.join(' ')}" if cask_conflicts.present?
+              f.puts "CASK_DEPENDENCIES=#{cask_dependencies&.join(' ')}" if cask_dependencies.present?
               f.puts "FORMULA_CONFLICTS=\"#{formula_conflicts&.join(" ")}\"" if formula_conflicts.present?
```

```suggestion
              f.puts "FORMULA_CONFLICTS=#{formula_conflicts&.join(" ")}" if formula_conflicts.present?
```

Looking at the later changes, presumably this should align with the others?